### PR TITLE
OP-846: Update Readme to include build and deploy instructions

### PR DESCRIPTION
**Pull request by lucashuang97**
Adding in instructions on how to build and deploy the docker container for future reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,15 @@ on default, the exporter will listen to `localhost:22222`
 
 # How to
 
-Install:
+Build:
+The Dockerfile is located here https://github.com/ecobee/epc/blob/main/docker/twemproxy_exporter/Dockerfile it will pull the main branch and build using it.
+ In the directory with the Dockerfile, run `docker build . -t gcr.io/platform-prod-618b7383/rke/twemproxy_exporter:<timestamp>`
```
Comment (randallrecobee) [inline README.md]: define the format of timestamp

File: README.md [hunk 2]
```diff
@@ -6,8 +6,15 @@ on default, the exporter will listen to `localhost:22222`
...
+
+Deploy:
+Manually push the built docker image to gcr using `docker push gcr.io/platform-prod-618b7383/rke/twemproxy_exporter:<timestamp>`, then update the tag wherever you are using the container (most still use "latest" so need to update to "timestamp")
```
Comment (randallrecobee) [inline README.md]: let's not add latest. focus on specific version

File: README.md [hunk 3]
```diff
@@ -6,8 +6,15 @@ on default, the exporter will listen to `localhost:22222`
 
 # How to
 
-Install:
+Build:
```
Comment (randallrecobee) [inline README.md]: nit: prepend '##'

**Review by AVos-ecobee — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -6,8 +6,15 @@ on default, the exporter will listen to `localhost:22222`
...
+The Dockerfile is located here https://github.com/ecobee/epc/blob/main/docker/twemproxy_exporter/Dockerfile it will pull the main branch and build using it.
+ In the directory with the Dockerfile, run `docker build . -t gcr.io/platform-prod-618b7383/rke/twemproxy_exporter:<timestamp>`
+
+Deploy:
+Manually push the built docker image to gcr using `docker push gcr.io/platform-prod-618b7383/rke/twemproxy_exporter:<timestamp>`, then update the tag wherever you are using the container (most still use "latest" so need to update to "timestamp")
+
+Install locally on machine:
```
Comment (randallrecobee) [inline README.md]: nit: prepend '##'

**Review by randallrecobee — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (lucashuang97) [reply README.md]: Yea I should make it clearer, lots of services use "latest" tag so we should use timestamp instead moving forward

**Review by randallrecobee — APPROVED** [review]

**Pull request merged**